docs/ubuntu_install.rst

```diff
@@ -34,18 +35,12 @@ Verify the ``SHA256SUMS`` file and move on to the next step if you see
 
     gpg --verify SHA256SUMS.gpg SHA256SUMS
 
-The next and final step is to verify the Ubuntu image. If you are using
-Linux, use the following command.
+The next and final step is to verify the Ubuntu image.
 
 ::
```

Comment:
Having the `::` on a separate line like this is an artifact from the automatic Markdown -> ReST conversion with Pandoc. You can move it up to the end of the previous line of text, e.g.

```rest
The next and final step is to verify the Ubuntu image. ::
```

I generally prefer this convention, because I think it makes the ReST source easier to read.